### PR TITLE
Remove the social fields also from the default-layout

### PR DESCRIPTION
**Pull request by Lyra2108**

### CODE REVIEW

File: inyoka_theme_default/templates/portal/profile.html [hunk 1]
```diff
@@ -108,67 +93,24 @@
...
-          <td class="hidden-info">
-            <a href="{{ user.jabber_url|e }}">{{ user.jabber|e }}</a>
-          </td>
-        {% else %}
-          <td>–</td>
-        {% endif %}
-        <td>{% if user.gpgkey %}0x{{ user.gpgkey|e }}{% else %}–{% endif %}</td>
-      </tr>
-      <tr>
-        <th colspan="3">{% trans %}Group memberships{% endtrans %} </th>
```
Comment (chris34) [inline inyoka_theme_default/templates/portal/profile.html:158]: why not showing the group membership(s) anymore?

File: inyoka_theme_default/templates/portal/profile.html [hunk 2]
```diff
@@ -108,67 +93,24 @@
...
-        <td>{{ user.wengophone|e or '–' }}</td>
-      </tr>
-      <tr>
-        <th>{% trans %}SIP{% endtrans %}</th>
-        <th>{% trans %}Jabber{% endtrans %}</th>
-        <th>{% trans %}GPG key{% endtrans %}</th>
-      </tr>
-      <tr>
-        <td>{{ user.sip|e or '–' }}</td>
-        {% if user.jabber and user.settings['show_jabber'] %}
```
Comment (chris34) [inline inyoka_theme_default/templates/portal/profile.html:146]: i know the user can't choose, whether everybody should see his jabber-id. However show it to the admins?

File: inyoka_theme_default/templates/portal/user_edit_profile.html [hunk 3]
```diff
@@ -43,10 +43,10 @@
...
 
     <h4>{% trans %}Contact addresses{% endtrans %}</h4>
-    {{ macros.inner_form(form, ['email', 'gpgkey', 'jabber', 'icq', 'msn', 'aim', 'yim', 'skype', 'wengophone', 'sip']) }}
+    {{ macros.inner_form(form, ['email', 'gpgkey', 'jabber']) }}
 
     <h4>{% trans %}Miscellaneous settings{% endtrans %}</h4>
-    {{ macros.inner_form(form, ['occupation', 'interests', 'website', 'launchpad', 'location']) }}
+    {{ macros.inner_form(form, ['website', 'launchpad', 'location']) }}
 
     {{ macros.input(form.signature) }}
```
Comment (chris34) [inline inyoka_theme_default/templates/portal/user_edit_profile.html]: add field for userpage here, too?

**Comment by chris34** [comment]
render the userpage [instead of (old) wiki-articles](https://github.com/Lyra2108/theme-default/blob/social-remove/inyoka_theme_default/templates/portal/profile.html#L118)? (as the edit-fields are added)

**Comment by Lyra2108** [comment]
I will open a separate PR for this, because it is a different issue.

**Pull request merged**